webhook_to_fedora_messaging/config/defaults.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+class Defaults:
+    DEBUG = True
+    TESTING = False
+    PROPAGATE_EXCEPTIONS = None
+    SECRET_KEY = "10505aec23775f6ecb61ded347654282d1be2c673e3b37b0372db6557f39f0ca"
+    PERMANENT_SESSION_LIFETIME = 604800
+    USE_X_SENDFILE = False
+    SERVER_NAME = None
```

Comment:
No need to redefine configuration values that are already default in Flask. This goes for most of the variables here.